Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG BASE_IMAGE=common-tools-base
+ARG RUNTIME_BASE_IMAGE
+ARG BINARY_PATH="/bin/bash"
+ARG EXPOSED_PORT=
+
+FROM ${BASE_IMAGE} as common-tools-monolith
+
+COPY . .
+
+# Build the project
+RUN make build_typescript
+RUN make build_rust
```

Comment:
Any reason why you don't want to do these builds in their own standalone builder that may be re-used for both `builder` and `runtime`?

Reply:
in theory. though I'd like to experiment more with that and more directly hooking up to the emitted files so docker can play nicer with Make's output detection